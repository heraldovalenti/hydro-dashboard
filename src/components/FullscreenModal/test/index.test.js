import React from 'react';
import FulscreenModal from '../';
import ShallowRenderer from 'react-test-renderer/shallow';

describe('<FulscreenModal/>', () => {
  it('default', () => {
    const Son = <div>Son</div>;

    const renderer = new ShallowRenderer();
    renderer.render(
      <FulscreenModal>
        <Son />
      </FulscreenModal>
    );
    const tree = renderer.getRenderOutput();
    expect(tree).toMatchSnapshot();
  });
});
