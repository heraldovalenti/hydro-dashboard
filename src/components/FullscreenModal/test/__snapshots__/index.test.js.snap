// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FulscreenModal/> default 1`] = `
<React.Fragment>
  <UNDEFINED
    isFullscreen={false}
    isInModal={false}
    toggleFullscreen={[Function]}
  />
  <ForwardRef(WithStyles)
    aria-labelledby="fullscreen-modal__ttitle"
    className="fullscreen-modal"
    fullScreen={true}
    onClose={[Function]}
    open={false}
  >
    <ForwardRef(WithStyles)
      id="fullscreen-modal__title"
      onClose={[Function]}
    />
    <ForwardRef(WithStyles)
      dividers={true}
    >
      <UNDEFINED
        isFullscreen={false}
        isInModal={true}
        toggleFullscreen={[Function]}
      />
    </ForwardRef(WithStyles)>
  </ForwardRef(WithStyles)>
</React.Fragment>
`;
